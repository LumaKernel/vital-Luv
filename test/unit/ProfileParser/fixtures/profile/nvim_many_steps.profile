SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_many_steps.vim
Sourced 1 time
Total time:   5.559638
 Self time:   5.559638

count  total (s)   self (s)
    1              0.000010 let x = 0
1000001              1.193184 for i in range(1000000)  " 1e6
1000000              1.282614   let x += 1
1000001              0.990176 endfor
    1              0.000012 echo x
                            

