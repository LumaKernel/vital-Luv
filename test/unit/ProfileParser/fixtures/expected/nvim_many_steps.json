[{"path":"/test/test_many_steps.vim","total_time":5.855752,"self_time":5.855752,"type":"script","lines":[{"self_time":1.8e-5,"count":1,"content":"let x = 0"},{"self_time":1.255137,"count":1000001,"content":"for i in range(1000000)  \" 1e6"},{"self_time":1.359367,"count":1000000,"content":"  let x += 1"},{"self_time":1.037135,"count":1000001,"content":"endfor"},{"self_time":2.3e-5,"count":1,"content":"echo x"},{"content":""}],"count":1}]
